shared/message/motion/KickCommand.proto

```diff
@@ -32,8 +32,11 @@ enum KickCommandType {
 }
 
 message KickCommand {
-    vec3            target            = 1;  // The point to kick
-    vec3            direction         = 2;  // force is the magnitude
+    /// The point to kick
```

Comment:
What?

Reply:
This was from a merge commit. This file isn't edited in this PR. 

In #312 The comment got moved from next to the field to above it with the triple slash, as you can see here.